staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -61,6 +61,8 @@ const (
 // Config contains the configuration for a given Cache.
 type Config struct {
 	// Maximum size of the history cached in memory.
+	//
+	// DEPRECATED: Cache capacity is dynamic and this field is no longer used.
 	CacheCapacity int
```

Comment:
Question, why we just don't simply delete the field?

Reply:
It's being set in some places. I would like to remove it, but in case we learn that something with this PR will not work over the next couple weeks, it will be easier to revert.